### PR TITLE
reinvent wrap up post

### PR DESCRIPTION
**Pull request by juhofriman**
tuommoista

### CODE REVIEW

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title: AWS re:invent 2017 wrap up
+author: juhofriman
+excerpt: Calling all the builders out there! You build it! You run it! And you are proud of it!
+tags:
+- AWS
+- re:Invent
+- cloud
+---
+
+Chaos. Chaos is the best word to describe this insanely large scale event taking place in a artificial city built in the middle of the desert - Las Vegas! At first, the event just felt bit too much for a small town boy like myself, but when you just free your mind and go with the flow, and don't even try to attend to all the sessions, you might just get your head up to the cloud.
```
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: in a artificial city --> in an artificial city

Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: small town --> small-town ?

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## All things secure
+
+Cloud security was a large topic at AWS re:invent 2017. Even mr. Vogel mentioned in his keynote, that every developer must become security engineer and that is something I can agree completely. When building solutions of any kind, you must take security as a first thing job, and whole team must understand solution end-to-end from the security perspective.
```
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: mr. Vogel --> Mr Vogel ?

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Your truly took part at the Security Jam hackathon, which was really neat sort of capture the flag competition. Teams we're given AWS accounts that had pre built solutions running in it with security flaws. Flaws needed to be fixed in order to get points on a road to glory. Unfortunately our random team didn't succeed that well, but we had really fun and educational session.
```
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: pre built --> pre-built

Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: Unfortunately our --> Unfortunately, our

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+AWS offers pretty nice array of security products, but there are loads of pretty impressive companies offering all kinds of cloud security solutions.
+
+**My top-5 AWS security hints**
+
+1. Check your S3-buckets once again and understand the security model of S3.
+2. Prepare for DDoS attacks, check out and understand how shield, WAF, route53 and cloudfront enables you to mitigate attacks.
```
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: Shield, WAF, Route53 and Cloudfront

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+You need to store the data and storing the date got really wide array of new options. [Serverless aurora](https://aws.amazon.com/blogs/aws/in-the-works-amazon-aurora-serverless/) is something that I'm super exited about. I love relational databases and now (well, in few months hopefully) I can just spin up some tables to the cloud instead of provisioning a full RDS instance. There's [Neptune](https://aws.amazon.com/neptune/) as a managed graph database. [DynamoDB global tables](https://aws.amazon.com/dynamodb/global-tables/). [DynamoBD backups](https://aws.amazon.com/dynamodb/backup-restore/) (yay! I really need this!), [Aurora multi-master](https://aws.amazon.com/about-aws/whats-new/2017/11/sign-up-for-the-preview-of-amazon-aurora-multi-master/)...
+
+You have the data, but you need tools to write the code to process the data, right? They announced [Cloud9](https://aws.amazon.com/cloud9), sort of cloud native IDE for developers. And it had the breakpoints for debugging live lambdas! How cool is that! To be honest, I'll stick with my weapon of choice IntelliJ IDEA, but for lambdas cloud9 is definately something to check out.
```
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: definately --> definitely

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+When you have the code, you need to deploy it. Whole smorgasbord of exiting new deployment opportunities was introduced for us - the docker freaks. Not interested in provisioning EC2 instances for you docker containers? Use [Fargate](https://aws.amazon.com/fargate/). Love kubernetes? Use [EKS](https://aws.amazon.com/eks/). There eventually will even be Fargate kubernetes!
+
+## All things chaos
+
+I mentioned chaos already. Chaos is something that we have in our systems all the time. When you go from monolith to microservice you introduce even more chaos. People have been really interested in the *chaos engineering* discipline, most notably introduced by Netflix. I had the priviledge to attend session by senior chaos engineer Nora Jones of Netflix talking how they do chaos engineering in Netflix scale. We have all heard about the open sourced [Chaos Monkey](https://github.com/Netflix/chaosmonkey), but I really feel that many people don't have that clear vision what chaos engineering actually is. Yes, she also was invited on stage in Vogel's keynote to talk about this.
```
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: priviledge --> privilege

File: _posts/2017-12-01-AWS-reinvent-wrap-up.md (hunk 3 above)
Comment (crystoll) [inline _posts/2017-12-01-AWS-reinvent-wrap-up.md]: Your truly --> Yours truly

Comment (crystoll) [reply _posts/2017-12-01-AWS-reinvent-wrap-up.md]: part at --> part of ?

Comment (juhofriman) [reply _posts/2017-12-01-AWS-reinvent-wrap-up.md]: part in taitaa olla oikein

Comment (crystoll) [reply _posts/2017-12-01-AWS-reinvent-wrap-up.md]: Jeba. Otetaan palanen sisässä.

**Pull request merged**